app/views/playlists/_item_list.html.erb

```diff
@@ -45,6 +45,13 @@ Unless required by applicable law or agreed to in writing, software distributed
     <%= @playlist.comment %>
   </div>
 <% end %>
+
+<div style="margin-bottom:.4em;">
```

Comment:
Again, not a huge deal but moving forward we should avoid in-lining styles in the current structure of this application, and create classes  ie. `<div class="my-awesome-buttons-wrapper">` or something hopefully re-usable and/or consistent with margin/padding definitions.